translations/create_translation_json.py

```diff
@@ -18,6 +19,11 @@
 TOOLBOX_NAME = "TOOLBOX_NAME"
 COGNITE = "__cognite__"
 
+LOCIZE_API_KEY = "9e22ecf1-1fe7-41f2-827b-9f51b7c67f8b"
```

Comment:
it is not recommended to check in these values even if this is a test. Environment variable should be used instead